src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver() {
```

Comment:
lets define toString() method to display drivers in exception messages(be attentive to sensitive data)